packages/now-cli/src/util/projects/link.ts

```diff
@@ -141,13 +141,27 @@ export async function getLinkedProject(
   }
 
   const spinner = output.spinner('Retrieving project…', 1000);
-  let org: Org | null;
-  let project: Project | ProjectNotFound | null;
+  let org: Org | null = null;
+  let project: Project | ProjectNotFound | null = null;
   try {
     [org, project] = await Promise.all([
       getOrgById(client, link.orgId),
       getProjectByIdOrName(client, link.projectId, link.orgId),
     ]);
+  } catch (err) {
+    if (err?.status === 403) {
+      spinner();
+      throw new NowBuildError({
+        message: `Could not retrieve Project Settings. To link your project again, run ${chalk.gray(
+          `\`rm -rf .vercel\``
```

Comment:
This won't work on Windows. We should probably render a different command when running under Windows.

Reply:
Because of the chalk styling, you mean? Should we just remove the colors instead?

Reply:
I updated the message to be more generic:

> Could not retrieve Project Settings. To link your project, remove the .vercel directory and deploy again